pkg/strace/tracer.go

```diff
@@ -111,6 +111,15 @@ type process struct {
 	lastSyscallStop *TraceRecord
 }
 
+// seccompEnabled is a flag indicating whether seccomp is enabled.
```

Comment:
By run, are you referring to `runLoop` (in `tracer.go`) or `run` in the sample code?

Also how does this look :
```go
type Process struct {
    // ... other fields
    seccompEnabled bool
    mu             sync.RWMutex
}
// EnableSeccomp 
func (p *Process) EnableSeccomp() {
    p.mu.Lock()
    defer p.mu.Unlock()
    p.seccompEnabled = true
}

// DisableSeccomp 
func (p *Process) DisableSeccomp() {
    p.mu.Lock()
    defer p.mu.Unlock()
    p.seccompEnabled = false
}

Reply:
nothing wrong with bringing in sync, we use it everywhere anyway, so no additional cost incurred, because of the busy box way in which we build the final binary.

If you use the atomic.Bool type and then the Load and Store methods, you don't need the your EnableSeccomp/DisableSeccomp functions and corresponding mutex.

```
type Process struct {
    // ... other fields
    SecComp atomic.Bool
}

if p.SecComp.Load() {}
p.SecComp.Store(true)
```

Reply:
Okay, this is better
But how do we update the `SecComp` in our sample code (considering process struct is `unexported`)